crates/bevy_animation/src/lib.rs

```diff
@@ -191,23 +190,12 @@ impl AnimationClip {
     }
 
     /// Add a [`VariableCurve`] to an [`EntityPath`].
-    pub fn add_curve_to_path(&mut self, path: EntityPath, curve: VariableCurve) {
+    pub fn add_curve_to_path(&mut self, bone_id: AnimationTargetId, curve: VariableCurve) {
```

Comment:
```suggestion
    pub fn add_curve_to_target(&mut self, bone_id: AnimationTargetId, curve: VariableCurve) {
```
Probably need to update this elsewhere too.

Reply:
Done.